### PR TITLE
[ECO-2513] Add explorer link to market page

### PR DESCRIPTION
**Pull request by CRBl69**
# Description

This PR adds a link to the coin page by clicking on the big emoji on the market page.

### CODE REVIEW

File: src/typescript/frontend/src/components/pages/emojicoin/components/main-info/MainInfo.tsx [hunk 1]
```diff
@@ -71,10 +81,12 @@ const MainInfo = ({ data }: MainInfoProps) => {
...
       >
-        <Emoji
+        <Link
           className="text-[24px] text-center md:display-2 my-auto text-white"
-          emojis={data.emojis}
-        />
+          href={explorerLink}
+        >
+          <Emoji emojis={data.emojis} />
+        </Link>
```
Comment (alnoki) [inline src/typescript/frontend/src/components/pages/emojicoin/components/main-info/MainInfo.tsx:89]: This link is great, is there also a way to make a link show up just below the clickable emoji too? Mainly because people might not realize immediately that it is clickable or copy-pastable

For example on https://emojicoin-dot-15qssceh7-econia-labs.vercel.app/market/laptop;high_voltage, just below the emoji at the top that is now clickable, there would be a link right below that looks something like:

[`0x43dc...eb67`](https://explorer.aptoslabs.com/coin/0x43dcf02dcc0f3759d00486052585bf1694acf85c7e3e7c4b4770c5216d58eb67::coin_factory::Emojicoin?network=testnet) [the window icon that indicates copy-paste]

Basically have it look as much as possible like the explorer

- [ ] Verify on mobile
- [ ] Verify on desktop
- [ ] Verify on other screens sizes are required

**Review by alnoki — CHANGES_REQUESTED** [review]

**Review by xbtmatt — APPROVED** [review]

**Review by xbtmatt — APPROVED** [review]
